.github/workflows/ci.yml

```diff
@@ -14,7 +14,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - uses: wagoid/commitlint-github-action@v4
```

Comment:
```suggestion
      uses: actions/checkout@v2
    - name: Validate commit message
      uses: wagoid/commitlint-github-action@v4
```

Reply:
nope, we need `fetch-depth: 0` because of the commit linter: https://github.com/wagoid/commitlint-github-action#usage

```
[](https://github.com/wagoid/commitlint-github-action#inputs)Note: It's necessary that you specify the fetch-depth argument to actions/checkout@v2 step. By default they fetch only latest commit of the branch, but we need more commits since we validate a range of commit messages.
```

Also, I updated the checkout GH action to V3 because I thought it was the latest stable release, but it seems it's not yet released as stable, so I'm reverting that.